packages/sdks/src/functions/transform-block.ts

```diff
@@ -0,0 +1,4 @@
+// Noope way for targets to make modifications to the block object if/as needed
+export function transformBlock(block: any) {
```

Comment:
```suggestion
export function transformBlock<T>(block: T): T {
```

Reply:
block will always be a BuilderElement but we need to make an update to the mitosis CLI to be able to support types wit the way we do transforms (will be tricky so just using `any` in the meantime). I can update with a note on this though

Reply:
found a better way, now uses the BuilderBlock type like other areas (IDE will complain about the import path from the overrides folder but that is a common issue)

we could move to using absolute imports come to think of it to fix the types issue. so anything in overrides imports from like `@/functions/foo` instead of `../functions/foo` (which often doesn't exist in the `overrides/*` folders so vscode will complain 

hadn't thought of that until just now. something we can do in the future to improve the DX cc @samijaber (tho will have to update the mitosis CLI stuff to handle so is no small job but can do sometime in the future when we have time)